Queries.md

```diff
@@ -0,0 +1,419 @@
+    FROM participants_who_have_a_channel_separated 
+    union
+    SELECT participant2 AS participant 
+    FROM participants_who_have_a_channel_separated 
+)
+```
+
+
+## AVG number of channels a node opened
+
+```sql
+with part AS (
+    SELECT  args->1#>'{hex}' AS participant1, args->2#>'{hex}' AS participant2, args->0#>'{num}' AS channel_id
+    FROM "event"
+    WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+)
+SELECT AVG(num_of_channels)
+FROM 
+(
+    SELECT participants, count(channel_id) AS num_of_channels
+    FROM 
+    (
+        SELECT participant1 AS participants, channel_id
+        FROM part
+        union
+        SELECT participant2 AS participants, channel_id
+        FROM part
+        order BY participants
+    ) part_table
+    GROUP BY participants
+) participants_data
+```
+
+
+## Nodes AND the amount of channels they opened
+
+```sql
+with part AS (
+    SELECT  args->1#>'{hex}' AS participant1, args->2#>'{hex}' AS participant2,  args->0#>'{num}' AS channel_id
+    FROM "event"
+    WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+)
+SELECT participants, channel_id
+    FROM 
+    (
+        SELECT participant1 AS participants, channel_id
+        FROM part
+        union
+        SELECT participant2 AS participants, channel_id
+        FROM part
+        order BY participants
+    ) part_table
+    GROUP BY participants
+```
+
+
+## Participant AND the specific channel he opened
+
+```sql
+with part AS (
+    SELECT  args->1#>'{hex}' AS participant1, args->2#>'{hex}' AS participant2,  args->0#>'{num}' AS channel_id
+    FROM "event"
+    WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+)
+SELECT participants, channel_id
+FROM 
+(
+	SELECT participant1 AS participants, channel_id
+	FROM part
+	union
+	SELECT participant2 AS participants, channel_id
+	FROM part
+	order BY participants
+) part_table
+```
+
+## Network capacity
+
+
+``Network capacity = sum of all deposited amounts for the channels that are still opened``
+
+```sql 
+Select sum(channel_capacity)
+FROM 
+(
+    SELECT (args->0#>'{num}')::numeric AS channel_id, (args->2#>'{num}')::numeric AS channel_capacity
+    FROM "event"
+    WHERE event = 'ChannelNewDeposit' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+) channels_and_their_deposits
+WHERE channel_id NOT IN 
+(
+    SELECT channel_id 
+    FROM
+    (
+        SELECT (args->0#>'{num}')::numeric AS channel_id 
+        FROM "event"
+        WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+    ) channel_opened_data
+    WHERE channel_id NOT IN 
+    (
+        SELECT (args->0#>'{num}')::numeric AS channel_id
+        FROM "event"
+        WHERE event = 'ChannelClosed' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+    )
+)
+```
+
+
+## The first three channels with the highest capacity
```

Comment:
**The first three channels with the highest capacity** Also a date histogram query.